src/index.html

```diff
@@ -1,17 +1,169 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <div class="group">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label class="personal">
+            Surname:
+            <input
+            type="text"
+            autocomplete="off"
+            name="Surname"
+            required
+            >
+          </label>
+          <label class="personal">
```

Comment:
add an empty line between `multline` sibling
```suggestion
          </label>
          
          <label class="personal">
```